src/pages/TrackGameFormPage.js

```diff
@@ -87,17 +89,36 @@ export default function AddGameFormPage({ onTrackGame }) {
         )}
         {step === 1 && (
           <TrackGameOne
-            onHandleChange={handleChangeNameOfGame}
+            onHandleChange={handleChangeInput}
             onAddFormFields={addFormFields}
             onRemoveFormFields={removeFormFields}
             nameOfGame={gameData.nameOfGame}
             players={players}
+            handleOnClickDot={nextStep}
+            handleOnClickBack={prevStep}
+          />
+        )}
+        {step === 2 && (
+          <TrackGameTwo
+            onHandleChange={handleChangePlayer}
+            players={players}
+            handleOnClickDot={nextStep}
+            handleOnClickBack={prevStep}
+          />
+        )}
+        {step === 3 && (
+          <TrackGameThree
+            onHandleChange={handleChangeInput}
+            notes={gameData.notes}
+            handleOnClickDot={nextStep}
+            handleOnClickBack={prevStep}
           />
         )}
-        {step === 2 && <TrackGameTwo onHandleChange={handleChangePlayer} players={players} />}
         <ButtonContainer>
-          {step !== 2 && <DefaultButton type="button" label="CONTINUE" onClick={nextStep} />}
-          {step === 2 && <PrimaryButton type="submit" label="CONFIRM" />}
+          {step !== 3 && (
+            <DefaultButton type="button" label="CONTINUE" onClick={() => nextStep(1)} />
+          )}
+          {step === 3 && <PrimaryButton type="submit" label="CONFIRM" />}
         </ButtonContainer>
```

Comment:
I like the idea of having different components / steps for the form., but there is a little bug.
If I don't enter a name in "step 2", I can continue to step 3 but can't submit (which I shouldn't).
Moreover there is no error message, so as a user I don't really know what went wrong.
I think it's because the `DefaultButton` is `type="button"` and not `submit` (but I am not quite sure about it...)